clang/lib/Sema/SemaSYCL.cpp

```diff
@@ -3517,7 +3524,8 @@ void Sema::MarkDevice(void) {
         case attr::Kind::SYCLIntelNoGlobalWorkOffset:
         case attr::Kind::SYCLIntelUseStallEnableClusters:
         case attr::Kind::SYCLIntelLoopFuse:
-        case attr::Kind::SYCLSimd: {
+        case attr::Kind::SYCLSimd:
+        case attr::Kind::SYCLIntelFPGAMaxConcurrency: {
```

Comment:
This should be 
case attr::Kind::SYCLIntelFPGAMaxConcurrency:
case attr::Kind::SYCLSimd: {